docs/io/unified.rst

```diff
@@ -278,6 +278,15 @@ incorporate the unit attribute of Quantity.
 Time
 ~~~~
 
+Astropy provides the following features for reading and writing ``Time``:
+
+- Writing and reading `~astropy.time.Time` Table columns to and from FITS tables
+- Reading time coordinate columns in FITS tables (not written by Astropy) as
```

Comment:
Here we need to be clear.  Astropy can read and write a well-defined subset of FITS standard time columns, namely those that store the time using a two-vector of JD times.  So Astropy can read certain FITS standard time columns even if they were not written by astropy (though it is not clear how common that actually is in the wild at this point).

Conversely astropy can read time coordinate columns in some common legacy FITS tables that pre-date the standard, e.g. Chandra, XMM and HST.  But someone could certainly write a file in such a format using astropy.

So I would suggest changing the language here.  Don't imply that we fully support the standard, and change from "non-Astropy written" to "legacy" or "legacy format".  I have not reviewed the other text but move away from distinguishing between "astropy-written" to what standards / conventions the file content complies with.